Sources/SwiftParserDiagnostics/SyntaxExtensions.swift

```diff
@@ -181,6 +181,15 @@ public extension TriviaPiece {
     default: return false
     }
   }
+
+  var isComment: Bool {
```

Comment:
Could we consider adding some documentation to clarify the purpose of this property?